sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/aggregator/Aggregator.java

```diff
@@ -15,23 +14,11 @@
 public interface Aggregator {
 
   /**
-   * Merges the current value into the given {@code aggregator} and resets the current value in this
+   * Returns the current value into as {@link Accumulation} and resets the current value in this
    * {@code Aggregator}.
-   *
-   * @param aggregator value to merge into.
-   */
-  void mergeToAndReset(Aggregator aggregator);
-
-  /**
-   * Returns the {@code Point} with the given properties and the value from this Aggregation.
-   *
-   * @param startEpochNanos the startEpochNanos for the {@code Point}.
-   * @param epochNanos the epochNanos for the {@code Point}.
-   * @param labels the labels for the {@code Point}.
-   * @return the {@code Point} with the value from this Aggregation.
    */
   @Nullable
-  Point toPoint(long startEpochNanos, long epochNanos, Labels labels);
+  Accumulation toAccumulationThenReset();
```

Comment:
maybe `accumulateAndReset()` ?

Reply:
I like `accumulate`, but I would use ThenReset because it is consistent with java stream APIs.